Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -267,6 +267,9 @@ public enum FeatureFlag: String, CaseIterable {
     /// activates the audio session in the background to avoid locks in the main thread
     case activateAudioSessionInBackground
 
+    /// Use cellular-specific network APIs instead of expensive network APIs
+    case useCellularNetworkAPIs
```

Comment:
Just a warning that this name of feature flag will be translated to `use_celullar_network_a_p_is` so maybe change to `Apis`